packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -153,19 +154,28 @@ export default {
         ? {
             ...this.$attrs,
             sizes: this.sizes,
-            width: this.width
-              ? this.width
-              : !this.srcset && console.error(`Missing required prop width.`),
-            height: this.height
-              ? this.height
-              : !this.srcset && console.error(`Missing required prop height.`),
             srcset: this.srcset,
           }
         : {
             ...this.$attrs,
             ...this.nuxtImgConfig,
           };
     },
+    styles() {
+      this.width
+        ? this.width
+        : !this.srcset && console.error(`Missing required prop width.`);
+      this.height
+        ? this.height
+        : !this.srcset && console.error(`Missing required prop height.`);
```

Comment:
I'm not sure that Conditional (ternary) operator for `width` and `height` are valid.
For me above `width` conditional evaluated to : 
```
if(this.width) {
 this.width
} else {
 !this.srcset && console.error(`Missing required prop width.`);
}
```
In my opinion below code is more readable. What do you think ? 
```
if ((!this.width && !this.srcset) || (!this.height && !this.srcset)) {
    return console.error(`Missing required prop height.`);
}
```

Reply:
Except it needs to show proper error depending on lacking prop - width or height so I'll modify it.

Reply:
I changed it and left it without return in conditions because I want both errors to appear and after all if it passes the condition with null values the with and height shouldn't appear in the end.